contracts/Governance/TokenRedeemer.sol

```diff
@@ -28,6 +29,32 @@ contract TokenRedeemer is ReentrancyGuard, Ownable2Step {
         _transferAll(underlying, destination);
     }
 
+    function redeemUnderlyingAndRepayBorrowBehalf(
```

Comment:
This contract is utilized in an ongoing proposal [here](https://app.venus.io/#/governance/proposal/266?chainId=56). Furthermore, this contract is for internal use exclusively, and all available functions are authorized by the owner. But, sure, we should provide NatSpec for all external functions, whether internal or external. It would be an extra work to cancel and re-propose the proposal, thus I think we should retain it as is. WDYT?